### ISSUE TITLE
[Issue]: Web source missing from filename

### ISSUE THREAD

**Issue by vevv** [MichaelYochpaz/iSubRip#53]
### Confirmations

- [X] I have checked the issues tab, and couldn't find an existing open issue for the issue I want to report.

### OS Type

Windows

### Python Version

3.10.6

### Package Version

2.5.0

### Description

Source in filename is "WEB", instead of "iT.WEB" as before.

Expected: `Witness.1985.iT.WEB.de-DE.vtt`
Got: `Witness.1985.WEB.de-DE.vtt`

### Output Log

```Text
$ isubrip https://tv.apple.com/us/movie/umc.cmc.4mk13hhxh4lamkkira2wjmv9j?l=en
Scraping 'https://tv.apple.com/us/movie/umc.cmc.4mk13hhxh4lamkkira2wjmv9j?l=en'...
Found movie: Witness [1985] (ID: 267841498)
Deutsch (Deutschland) (de-DE) subtitles were successfully downloaded.
1/1 matching subtitles have been successfully downloaded.
```

**Comment by MichaelYochpaz** [comment]
You're right. Pushed a fix on a5156f31fd3ab39fab61d303f9ef9895660a2bf3.
Will be included on the next update.

**Comment by MichaelYochpaz** [comment]
Fixed on the new [v2.5.1 update](https://github.com/MichaelYochpaz/iSubRip/releases/tag/v2.5.1) :)
